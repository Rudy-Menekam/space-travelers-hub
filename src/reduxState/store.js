import { configureStore } from '@reduxjs/toolkit';
import missionReducer from './missionSlice';
import rocketReducer from './rockets/rocketSlice';

const store = configureStore({
  reducer: {
    missions: missionReducer,
    Rocket: rocketReducer,
  },
});
export default store;
