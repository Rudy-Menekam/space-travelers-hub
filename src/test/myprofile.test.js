import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../reduxState/store';
import Profile from '../components/my_profile';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <React.StrictMode>
        <Provider store={store}>
          <BrowserRouter>
            <Profile />
          </BrowserRouter>
        </Provider>
      </React.StrictMode>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
