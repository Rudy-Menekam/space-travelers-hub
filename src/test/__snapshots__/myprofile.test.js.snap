// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="missionProfile"
>
  <div
    className="missionContainer"
  >
    <h1>
      My Missions
    </h1>
    <ul />
  </div>
  <div>
    <div
      className="rocket_filter"
    >
      <h1>
        My Rockets
      </h1>
      <ul>
        <li>
          No rockets have been reserved.
        </li>
      </ul>
    </div>
  </div>
</div>
`;
